packages/react/src/components/Storage/FileUploader/hooks/useFileUploader/__tests__/useFileUploader.test.tsx

```diff
@@ -85,11 +85,11 @@ describe('useFileUploader', () => {
     expect(result.current.fileStatuses.length).toEqual(1);
   });
 
-  it('returns 1 and updates file statuses when not hasMultipleFiles and target length is 1', () => {
+  it('returns 1 and updates file statuses when not allowMultipleFiles and target length is 1', () => {
```

Comment:
Off subject: Can we make a note to clean up these test descriptions

Reply:
Yes, will address in a subsequent pr